src/OSPSuite.Presentation/Presenters/Importer/ImporterPresenter.cs

```diff
@@ -216,18 +216,22 @@ private void loadSheets(IDataSourceFile dataSourceFile, Cache<string, DataSheet>
             return;
          }
 
-         var mappings = dataSourceFile.Format.Parameters.OfType<MetaDataFormatParameter>().Where(p => p.ColumnName != null).Select(md => new MetaDataMappingConverter()
-         {
-            Id = md.MetaDataId,
-            Index = sheetName => md.IsColumn ? dataSourceFile.DataSheets[sheetName].RawData.GetColumnDescription(md.ColumnName).Index : -1
-         }).Union
+         var dataMappings = dataSourceFile.Format.Parameters.OfType<MetaDataFormatParameter>().Where(p => p.ColumnName != null).Select(md =>
+            new MetaDataMappingConverter()
+            {
+               Id = md.MetaDataId,
+               Index = sheetName => md.IsColumn ? dataSourceFile.DataSheets[sheetName].RawData.GetColumnDescription(md.ColumnName).Index : -1
+            }).ToList();
```

Comment:
no need for a tolist here since you are going to create a union right away after

Reply:
Main reason for this is that without the ToList() Resharper gives me a warning that there is a possible multiple enumeration. So I thought it is safer this way. But again it can easily be that I am missing something...

Reply:
ok